src/components/manage/Blocks/Table/View.jsx

```diff
@@ -17,7 +17,7 @@ import { settings } from '~/config';
  * @extends Component
  */
 const View = ({ data }) => (
-  <p className="documentDescription">
+  <p>
```

Comment:
Why is the <table> in a <p> tag? According to the HTML5 spec <p> will be closed automatically at the beginning of <table>. The following </p> will yield "Error: No p element in scope but a p end tag seen." on https://validator.w3.org